mlflow/models/auth_policy.py

```diff
@@ -0,0 +1,75 @@
+from typing import Optional
+
+from mlflow.models.resources import Resource, _ResourceBuilder
+
+
+class UserAuthPolicy:
```

Comment:
```suggestion

@experimental
class UserAuthPolicy:
```

Let's add `@experimental` marker for all new classes/apis.